crawler.py

```diff
@@ -96,6 +98,10 @@ def do_something(driver):
         random.shuffle(children)
 
         try:
+            # If there are some remaining elements that are not ignored, prefer them
```

Comment:
If we do this, we are actually only using the elements on which we haven't clicked yet.
We should first try with set(children) - already_clicked_elems, and if we can't find any element then we should use one from already_clicked_elems.

Reply:
Oh sorry, I had missed the condition! Please also add an explanatory comment (something like "If we have clickable elements on which we haven't clicked yet, use them; otherwise, use all elements")